clusterware/clusterware-benchware/package/build.sh

```diff
@@ -2,7 +2,7 @@
 
 package_name='clusterware-benchware'
 
-VERSION="2018.1.1"
+VERSION="2018.1.2"
```

Comment:
Can we DRY up the setting of the version? Otherwise someone is going to forget to bump it is both location. Instead the `config.yaml` can be parsed to pull out the version number